Documentation/contributing/development/dev_setup.rst

```diff
@@ -601,6 +601,30 @@ Patch version
 
 #. Submit all your changes into a new PR.
 
+Making changes to the Helm chart
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Helm chart is located in the ``install/kubernetes`` directory. The
+``values.yaml.tmpl`` file contains the values for the Helm chart which are being used into the ``values.yaml`` file.
+
+When updating or adding a value they can be synced to the ``values.yaml`` file by running the following command:
+
+.. code-block:: shell-session
+
+   $ make -C install/kubernetes cilium/values.yaml
+
+Before submitting the changes the ``README.md`` file needs to be updated, this can be done using the ``docs`` target:
+.. code-block:: shell-session
+
+   $ make -C install/kubernetes docs
+
+At last you might want to check the chart using the ``lint`` target:
+.. code-block:: shell-session
+
+   $ make -C install/kubernetes lint
+
+Your change to the Helm chart is now ready to be PRed!
```

Comment:
The spell checker fails to recognise `PRed`, can you please use something more standard? If you really want to keep the term, you have to add it to the list of spelling exceptions, see https://github.com/cilium/cilium/actions/runs/4174769460/jobs/7228777718